src/OrbitGl/CoreMath.h

```diff
@@ -36,18 +36,20 @@ struct ClosedInterval {
     return this->min <= closed_interval.max && this->max >= closed_interval.min;
   }
 
+  [[nodiscard]] bool Contains(float value) { return this->min <= value && this->max >= value; }
+
+  bool operator==(const ClosedInterval& interval) const {
```

Comment:
Can you use the hidden friend idiom?

Reply:
Nit: And nodiscard.

Reply:
And please also define operator!= by calling operator==. They should always come in pairs.